### PR TITLE
Use sqlite3 library.

### PR DESCRIPTION
**Pull request by hiltonc**

### CODE REVIEW

**Comment by hiltonc** [comment]
These are the changes I needed to make to get this to work with LDSContent. You can see the corresponding code at https://github.com/CrossWaterBridge/LDSContent/tree/fts.

If these aren't the changes you want to make, that's OK. I just wanted to share what I was able to get to work with you.

**Comment by hiltonc** [comment]
I merged the `fts` branch into `master`, so you'll want to look at that instead.

File: FTS3HTMLTokenizer.podspec [hunk 1]
```diff
@@ -1,21 +1,19 @@
 Pod::Spec.new do |s|
   s.name         = 'FTS3HTMLTokenizer'
-  s.version      = '1.1'
+  s.version      = '1.1.1'
```
Comment (stephanheilner) [inline FTS3HTMLTokenizer.podspec]: Can you make this `2.0.0`?

**Comment by stephanheilner** [comment]
Looks good, can you bump the version to 2.0.0, it wouldn't let me add to your commit.

File: FTS3HTMLTokenizer.podspec (hunk 1 above)
Comment (hiltonc) [inline FTS3HTMLTokenizer.podspec]: Yup!

**Pull request merged**